docs/zq/functions/has.md

```diff
@@ -0,0 +1,48 @@
+### Function
+
+&emsp; **has** &mdash; test a value's existence
```

Comment:
```suggestion
&emsp; **has** &mdash; test values' existence
```

Reply:
This voice isn't parallel with the rest of reference.md

Reply:
True. What I was getting at is that this accepts multiple arguments, not just one, but the salience of that point is marginal.

Reply:
Oh yes of course.